os/logm/logm_get.c

```diff
@@ -18,21 +18,16 @@
 #include <tinyara/logm.h>
 #include "logm.h"
 
-volatile int new_logm_bufsize = 0;
-
 /* This will be moved to upper layer or changed for protected build  */
 /* Refer logm_param_type_e for getparam values */
-int logm_get(enum logm_param_type_e type, void *value)
+int logm_get(enum logm_param_type_e type, int *value)
```

Comment:
As I  remember, I already gave review regarding this.
logm_get is not good name.
module_action_purpose is better.
In this case logm_get_something.